spec/components/schemas/Payments/ResponseSources/06_PaymentResponsePayPalSource.yaml

```
@@ -0,0 +1,9 @@
+type: object
+description: A PayPal payment source
+allOf:
+  - $ref: "#/components/schemas/PaymentResponseSource"
+  - type: object
+    properties:
+      reference:
+        type: string
+        description: The unique payment reference identifer from the payment request
```

~identifer~ *identifier*

---

@philippe-leonhardt-cko 
```suggestion
        description: The unique payment reference identifier from the payment request
```
Is this good enough? Or do you think this could be more explicit, as with the request source

---

That's good enough in the response.